.pre-commit

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
Here the hook is a bit advanced :blush: It checks only changed/staged .js files instead of everything